tests/playwright/src/specs/image-manifest.spec.ts

```diff
@@ -0,0 +1,76 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import { ArchitectureType } from '../model/core/platforms';
+import { expect as playExpect, test } from '../utility/fixtures';
+import { waitForPodmanMachineStartup } from '../utility/wait';
+
+const architectures: string[] = [ArchitectureType.AMD64, ArchitectureType.ARM64];
+const manifestName: string = 'localhost/build-image-test (manifest)';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+test.beforeAll(async ({ runner, welcomePage, page }) => {
+  runner.setVideoAndTraceName('pull-image-e2e');
+
+  await welcomePage.handleWelcomePage(true);
+  await waitForPodmanMachineStartup(page);
+});
+
+test.afterAll(async ({ runner }) => {
+  await runner.close();
+});
+
+test('Image Manifest e2e Validation', async ({ navigationBar }) => {
+  let imagesPage = await navigationBar.openImages();
+  await playExpect(imagesPage.heading).toBeVisible();
+
+  const buildImagePage = await imagesPage.openBuildImage();
```

Comment:
I would divide the test into multiple test cases, ie. build the image using cross-arch build, check manifest details, delete, for example.